ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -51,7 +51,7 @@ import { VulnerabilityState } from 'types/cve.proto';
 import AdvancedFiltersToolbar from 'Containers/Vulnerabilities/components/AdvancedFiltersToolbar';
 import LinkShim from 'Components/PatternFly/LinkShim';
 
-import { createFilterTracker } from 'Containers/Vulnerabilities/utils/telemetry';
+import { createFilterTracker } from 'utils/telemetry';
```

Comment:
Aha, since this contribution touches all of the `import` statements, what do you think are pro and con of analytics.ts as file name?

For comparison:
* analytics: src/global/initializeAnalytics.js and src/hooks/useAnalytics.ts
* telemetry: src/reducers/telemetryConfig.js and src/services/TelemetryConfigService.ts

Reply:
I don't have a strong opinion on this, especially since it seems we are already 50/50. It would be nice if we had a unified term though. Unfortunately it looks like the segment library uses the term 'analytics' and our backend services use the term 'telemetry', so there isn't a clear pull from either of those directions.

Reply:
analyticsEventTracking sounds precise, nice!